docs/setup-and-installing.md

```diff
@@ -1,153 +1,81 @@
-
-> **It’s done when:** You see `spatial upload MyGDKAssembly succeeded` printed in your console output.
-
-1. Launch a cloud deployment
-<br>The command `spatial cloud launch` deploys a project to the cloud. Its full syntax is:
-
-    `spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>`
-
-    where:
-
-    - `<assembly name>` is the name of the assembly the deployment will use (the one you named in step 3).
-    - `<launch configuration>` is the configuration file for the deployment. This project includes one called `default_launch.json`.
-    - `<deployment name>` is a name of your choice, which you’ll use to identify the deployment. This must be in lowercase.
-    - `<snapshot file>` is the snapshot of the world you want to start from. This project includes one called `snapshots/default.snapshot`.
-
-    In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
-
-    This command defaults to deploying to clusters located in the US. So if you’re in Europe, add the `--cluster_region=eu` flag for better performance.
-
-> **It’s done when:** You see `Successfully created deployment` printed in your console output.
-
-
-1. Launch a game client:
-    1. Open https://console.improbable.io/projects.
-    <br>You’ll see the project and the deployment you just created.
-
-    1. Click the deployment’s name to open the overview page.
-
-    1. To get links to share the game with others, click **Share**.
-
-    1. Click **Launch**.
-        > You can ignore the prompt to install the Launcher, as it’s installed as part of SpatialOS.
-
-    1. Once you’ve finished playing, click **Stop** in the Console.
+### Sign up for a SpatialOS account
+
+You'll need a SpatialOS account in order to run the SpatialOS GDK for Unity.
+Follow [this link](https://improbable.io/get-spatialos) to sign up for free.
+
+### Set up your machine
+
+#### Windows
+
+1. [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive) 
+    - Make sure to select the following components during installation:
+        - **Linux Build Support**
+        - **Mac Build Support**
+2. [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Verified with version `15.8.4`) or [Rider](https://www.jetbrains.com/rider/) (Verified with version `2018.2.1`)
+    - If you are using Visual Studio, within the Visual Studio Installer, on the **Workloads** tab:
+        - Select **Game development with Unity** 
+        - Select **.NET Core cross-platform development** 
+        - In the **Summary** on the right: 
+            - Deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.8)
+            - Make sure **Visual Studio Tools for Unity** is selected
+    - If you are using Rider: 
+        - Install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience
```

Comment:
nit: Unsure why **Unity Support** is bolded here.

Reply:
It follows the pattern of the other items in the list